volatility3/framework/constants/_version.py

```diff
@@ -1,7 +1,7 @@
 # We use the SemVer 2.0.0 versioning scheme
 VERSION_MAJOR = 2  # Number of releases of the library with a breaking change
 VERSION_MINOR = 26  # Number of changes that only add to the interface
-VERSION_PATCH = 1  # Number of changes that do not change the interface
+VERSION_PATCH = 2  # Number of changes that do not change the interface
```

Comment:
This can stay as a PATCH bump, because it just means the framework change internally, rather than anything that affects any of the APIs in it.